app/device_mac.py

```
@@ -40,7 +40,7 @@ def lookup_ip_via_arp_for_mac(mac: str) -> str | None:
         return None
     try:
         completed = subprocess.run(
-            ["arp", "-a"],
+            ["arp", "-an"],
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)